doc/source/theory/dem/dem.rst

```diff
@@ -82,26 +82,26 @@ The relative velocities calculated to allow updating the tangential overlap are
      - Linear model definitions
      - Nonlinear viscoelastic model definitions
    * - Normal spring constant
-     - :math:`k_n = \frac{16}{15}\sqrt{R_{e}}Y_{e}\left(\frac{15m_{e}V^2}{16\sqrt{R_{e}}Y_{e}}\right)^{0.2}`
+     - :math:`k_n = 1.2024(m^{0.5}E)Y_{e}^{2}R_{e}\nu_{e})^{0.4}`
```

Comment:
I would add the reference in the .rst file for the formula.

Reply:
We don't use this equation anymore.